src/datasets/utils/resources/languages.json

```diff
@@ -670,6 +670,7 @@
     "ja-Latn": "Japanese (Latin)",
     "jak": "Jakun",
     "jam": "Jamaican Patois",
+    "japh1234": "Japhug",
```

Comment:
For the moment, we are not using Glottolog codes in our `languages.json` list. We use IANA language codes.

If you would like to propose another approach, it would be better to open an Issue page (or participate in some already opened, like #4881) to discuss and agree on it before being included to the library. Thanks.

Reply:
Yes, I noticed it was checked, that is why I modified the corpus card to include both ISO 639-3 and Glottolog codes (the latter are used as sub-corpus names)

Reply:
So, if I understand well, I should only add the IANA/ISO 639-3 codes in this file, and the Glottolog codes in my corpus `language_details` field, right?

Reply:
Yes, for the moment I think that would be the simplest/fastest thing...

Reply:
The change of the coding schema has to be agreed with the rest of the teams and will take some time...